### PR TITLE
fix keras 4d softmax wrong result issue

### PR DESCRIPTION
**Pull request by ericwu155214**
Hi,

I got a wrong result issue in case of fused softmax node in conv,
here is the solution.

please check it.
thanks.

### CODE REVIEW

**Comment by MrWhoami** [comment]
I don't understand why we need to do the transpose and change the axis attribute at the same time. I think after adding transpose before and after the Softmax, the axis do not need to be changed.

**Comment by ericwu155214** [comment]
> I don't understand why we need to do the transpose and change the axis attribute at the same time. I think after adding transpose before and after the Softmax, the axis do not need to be changed.

The axis attribute actually will not be changed after doing transpose. Axis value is only changed in the case of "do_transpose" is False.

**Review by MrWhoami — APPROVED** [review]
